bitnami/zookeeper/Chart.yaml

```
@@ -21,4 +21,4 @@ name: zookeeper
 sources:
   - https://github.com/bitnami/bitnami-docker-zookeeper
   - https://zookeeper.apache.org/
-version: 7.0.1
+version: 7.0.2
```

```suggestion
version: 7.0.3
```
A new version was released some hours ago, see 620b1a186fa. Another version bump is needed